environment.yml

```diff
@@ -0,0 +1,10 @@
+name: vector
+channels:
+  - conda-forge
+dependencies:
+  - jupyterlab
+  - pip
+  - pytest
+  - root >=6.18.04
```

Comment:
I guess you're adding ROOT here for tests rather than for the normal user. I think that's fine, at least for now.

Reply:
A normal user should do pip install vector. And for the eventual binder examples, there will probably be a comparison with ROOT.